s5_subwords/Readme.md

```diff
@@ -2,60 +2,41 @@
 
-We use the Málrómur speech data [1] for training the models. Please find instructions on preparing the speech data below. Instead of implementing the subword algorithms from scratch, as is done with BPE in the s5c gale-arabic recipe, we provide pre-processed Icelandic training data (Málrómur transcripts) for three different subword segmentation methods: BPE, Kvistur, and Unigram (SentencePiece). Please refer to the [technical report](https://github.com/svanhviti16/subword-asr-icelandic/blob/master/s5/Subword_modelling_ASR_summer_2020.pdf) for details.
+We use the Málrómur speech data [1] for training the models. Please find instructions on preparing the speech data below. Please refer to the [technical report](https://github.com/svanhviti16/subword-asr-icelandic/blob/master/s5/Subword_modelling_ASR_summer_2020.pdf) for details.
 
-## Kaldi and basic setup
+### Acustic data
+To do... 
 
-The system uses the Kaldi speech recognition toolkit. Find download and installing instructions in Kaldi's online documentation:
-http://www.danielpovey.com/kaldi-docs/install.html or https://github.com/kaldi-asr/kaldi. We are using a version of Kaldi from February 2017.
-A thorough documentation of Kaldi can be found at http://kaldi-asr.org/, for a step-by-step tutorial for beginners see http://kaldi-asr.org/doc/kaldi_for_dummies.html.
 
-*Note:* Per default Kaldi will be installed creating debuggable binaries at the cost of speed. When optimizing for speed you should edit `kaldi/src/kaldi.mk` as described
-in http://kaldi-asr.org/doc/build_setup.html.
+### Subword methods
+We provide scripts for three subwords methods. We modified gthe ale-arbic Byte Pair endcoding recipe in the Kaldi repository for one of the BPE implementations and used Google’s SentencePiece (SP) library for the other BPE implementation (SP_BPE) and the Unigram algorithm. We also have the package Morfessor. 
 
-Kaldi has an excellent collection of examples in it's `$KALDI_ROOT/egs/` directory, have a look at `egs/wsj/` for example.
+```
+local
+└── sw_methods
+	├──bpe
+	├──sp
+	└──morfessor
+```
 
-When you have installed Kaldi and cloned this project, edit `path.sh` and `setup.sh` according to the location of your Kaldi installation.
-Run `setup.sh` from your `s5` directory to create symlinks to the folders `steps` and `utils` from an example project. 
+### Dependacies (other than Kaldi)
+For Kaldi installtion help refer to ../documentation/setup_kaldi.md
 
-Now you should be all set up for experimenting with Kaldi, below are descriptions on both how to use existing models and how to train your own.
+* Python 3.x
 
-## Icelandic speech data
-
-On http://malfong.is/ you can find the Málrómur data used for training these models. Please download this data, and only use the utterances marked as 'correct'.
-
```

Comment:
Don't you need to include the information on where to get the data? Or are you going to write that in Acoustic data?